docs/docsite/rst/inventory/implicit_localhost.rst

```diff
@@ -0,0 +1,34 @@
+.. _implicit_localhost:
+
+Implicit 'localhost'
+====================
+
+When you try to use a 'localhost' and you don't have it defined in inventory, Ansible will create an implicit one for you.::
+
+    - hosts: all
+      tasks:
+        - name: check that i have log file for all hosts on my local machine
+          stat: path=/var/log/hosts/{{inventory_hostname}}.log
```

Comment:
I know this isn't the point of the task, though we say you should always specify the `state`
`state: present`

Reply:
except ... `stat` has no `state` option

Reply:
oh, miss read, ignore